controllers/configurationpolicy_utils.go

```diff
@@ -65,6 +65,46 @@ func addRelatedObjects(
 	return relatedObjects
 }
 
+// addCondensedRelatedObjs does not include all of relatedObjs.
+// The Name field is "*". The list of objects will be presented on the console.
+// This function is used when it is "musthave", object-template doesn't have "name" of metadata and NonCompliant.
+func addCondensedRelatedObjs(
+	rsrc schema.GroupVersionResource,
+	compliant bool,
+	kind string,
+	namespace string,
+	namespaced bool,
+	reason string,
+) (relatedObjects []policyv1.RelatedObject) {
+	// Initialize the related object from the object handling
+	var relatedObject policyv1.RelatedObject
+
+	if compliant {
+		relatedObject.Compliant = string(policyv1.Compliant)
```

Comment:
The description of this function indicates you shouldn't call this if it's compliant. So should this code not exist in the function?